web/src/components/atoms/Input.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+interface IInputProps {
+  title: string
+  type: string
+  placeholder: string
+  onChange(value: any): void
+}
+
+const Input = ({ title, type, placeholder, onChange }: IInputProps) => {
```

Comment:
```suggestion
const Input: FC<Props> = ({ title, type, placeholder, onChange }) => (
```

これFCで書いてない理由あります？なければこの感じのほうが良いかなと思います